src/instructlab/model/download.py

```diff
@@ -149,35 +159,35 @@ def _build_oci_model_file_map(self, oci_model_path: str) -> dict:
         index_hash = ""
         index_ref_path = f"{oci_model_path}/index.json"
         try:
-            index_ref = _load_json(Path(index_ref_path))
+            index_ref = self._load_json(Path(index_ref_path))
             match = None
             for manifest in index_ref["manifests"]:
                 if (
                     manifest["mediaType"]
                     == "application/vnd.oci.image.manifest.v1+json"
                 ):
-                    match = _extract_SHA(manifest["digest"])
+                    match = self._extract_SHA(manifest["digest"])
+                    break
 
             if match:
                 index_hash = match.group(1)
             else:
                 raise ValueError(
-                    f"could not find hash for index file at: {oci_model_path}"
+                    f"failed to find image hash in the index file: {oci_model_path}"
```

Comment:
This is more for my learning, but is it correct to refer to this file as the "image"? It references one of the blobs that represents the manifest that contains the mapping between other blobs and their respective filenames

Reply:
Oh that may well be that I screwed this one up. The intent here was to say it's not a hash *for* but *in* the index file. The `image` part is wrong - I will drop it.